### PR TITLE
confirmation of default interval and splay when using -d

### PR DESCRIPTION
**Pull request by JMccProgress**
confirmation of default interval and splay when using -d    - change prompted by customer support ticket.

### Description

adds further information about the use of daemonized chef-client 

### Definition of Done

n/a

### Issues Resolved

this was prompted by customer support ticket 27499

### Check List

- [x] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

Signed-off-by: James McCormick <James.McCormick@progress.com>

### CODE REVIEW

**Comment by IanMadd** [comment]
Hi @JMccProgress, can you add your DCO signoff. 

https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco

File: content/ctl_chef_client.md [hunk 1]
```diff
@@ -78,6 +78,9 @@ This command has the following options:
 :   Run the executable as a daemon. Use `SECONDS` to specify the number
     of seconds to wait before the first daemonized Chef Infra Client
     run. `SECONDS` is set to `0` by default.
+    
+    If not otherwise set, this will also make use of the default `--interval`
```
Comment (kagarmoe) [inline content/ctl_chef_client.md]:
```suggestion
    Left unset, the daemon uses the default `--interval` and `--splay` values.
```

File: content/ctl_chef_client.md [hunk 2]
```diff
@@ -78,6 +78,9 @@ This command has the following options:
 :   Run the executable as a daemon. Use `SECONDS` to specify the number
     of seconds to wait before the first daemonized Chef Infra Client
     run. `SECONDS` is set to `0` by default.
+    
+    If not otherwise set, this will also make use of the default `--interval` 
+    and `--splay` values.
```
Comment (kagarmoe) [inline content/ctl_chef_client.md]:
```suggestion
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]
Small change.

**Review by kagarmoe — APPROVED** [review]
tenor-240764314

**Comment by kagarmoe** [comment]
Closed in favor of https://github.com/chef/chef-web-docs/pull/3324 (with DCO)

**Pull request closed without merging**